src/LibVLCSharp/Platforms/Windows/VideoViewBase.cs

```diff
@@ -89,6 +89,21 @@ protected override void OnApplyTemplate()
 
         }
 
+        /// <summary>
+        /// Clears the current view restoring the initial visual state.
```

Comment:
For future reference, I think it would be good to say this is a libvlcsharp workaround for https://code.videolan.org/videolan/vlc/-/issues/23667 and it is currently windows-specific.